crypto/ec/build.info

```diff
@@ -139,3 +147,8 @@ IF[{- !$disabled{'sm2'} -}]
   GENERATE[ecp_sm2p256-armv8.S]=asm/ecp_sm2p256-armv8.pl
   INCLUDE[ecp_sm2p256-armv8.o]=..
 ENDIF
+
+IF[{- !$disabled{'sm2'} -}]
+  GENERATE[ecp_sm2p256-riscv64.S]=asm/ecp_sm2p256-riscv64.pl
+  INCLUDE[ecp_sm2p256-riscv64.o]=..
+ENDIF
```

Comment:
Please add the missing EOL at EOF

Reply:
@t8m I have updated the commit, please re-review it